docs/pages/versions/unversioned/sdk/splash-screen.mdx

```diff
@@ -80,6 +85,116 @@ export default function App() {
 }
 ```
 
+## Configuration
+
+### Expo config
+
+To configure `expo-splash-screen`, use the built-in [config plugin](/guides/config-plugins) in the Expo config (**app.json** or **app.config.js**) for [EAS Build](/build/introduction) or with `npx expo run:[android|ios]`. The plugin allows you to configure the following properties that cannot be set at runtime and require building a new app binary to take effect:
+
+<ConfigPluginProperties
+  properties={[
+    {
+      name: '<platform>.backgroundColor',
+      description: 'Color to fill the splash screen screen background.',
+    },
+    {
+      name: '<platform>.resizeMode',
+      default: 'contain',
+      description: 'Resize mode to render the splash screen image on a device, must be one of "cover", "contain" or "native".',
+    },
+    {
+      name: '<platform>.image',
+      description:
+        'Image to render on the splash screen, must be a png file. Both local paths or remote URLs are supported.',
```

Comment:
```suggestion
        'Image to render on the splash screen, must be a PNG file. Both local paths or remote URLs are supported.',
```

There is an inconsistency in this table for using "png". Let's stick to "PNG file" as per our [writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#abbreviations) or just .png.